tests/Functional/CRUDTest.php

```diff
@@ -79,7 +80,9 @@ public function testEdit(): void
 
     public function testDelete(): void
     {
-        $entityManager = static::bootKernel()->getContainer()->get('doctrine')->getManager();
+        $doctrine = static::bootKernel()->getContainer()->get('doctrine');
```

Comment:
I think the container resolution can be solved warming up cache before running phpstan. (can be done executing test previously). Then you add the xml to the phpstan config and it can solve those issues